src/main/java/ch/admin/bag/covidcertificate/domain/KpiData.java

```diff
@@ -29,11 +31,13 @@ public KpiData(LocalDateTime timestamp, String type, String value, String uvci,
         this.uvci = uvci;
         this.details = details;
         this.country = country;
+        this.systemSource = "UI";
```

Comment:
I personally prefer to add SystemSource as additional parameter and add it to the method call where it is used. If thats the case at the "definitive" edition this may stay as is for the moment.

Reply:
It will be set with an additional parameter in the final version (also: there will be the vaues API and UI later).
For now, I wanted to make clear that _currently_ it is always "UI".